index.html

```diff
@@ -2428,10 +2429,33 @@ <h3 id="Interfaces-Events">Events</h3>
 <code class="js">knex.select('*')
   .from('users')
   .on('query', function(data) {
-    app.log(data);
+  app.log(data);
```

Comment:
Does example still look correct in browser? looks like indentation was removed here by an accident.